aquia-inc/template-vite-react
# build: reduce node engine version specificity to major

## Summary

Reduces the specificity of the Node.js version lock in .nvmrc and in the `engines.node` in package.json from `20.11.0` to `20`. This improves compatibility for developers' environments as long as v20 is used.

### Added

- n/a

### Changed

- `engines.node` in package.json
- .nvmrc

### Deprecated

_n/a_

### Removed

_n/a_

### Fixed

_n/a_

## How to test

First, test .nvmrc (ensure nvm is installed):

```
nvm install
nvm use
```

Then, verify that installation succeeds with the right node engine:

```
yarn install
```
